src/frontends/pytorch/src/frontend.cpp

```diff
@@ -119,7 +119,8 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
 }
 
 void FrontEnd::add_extension(const std::shared_ptr<ov::Extension>& extension) {
-    FRONT_END_NOT_IMPLEMENTED(add_extension);
+    // Extension loading mechanism is not implemented, any extensions will be ignored
```

Comment:
Can you please create a ticket and JIRA about adding extensions for PT FE and list it here

Reply:
added ticket number